recipes/pybind11-rdp/meta.yaml

```
@@ -0,0 +1,48 @@
+package:
+  name: pybind11-rdp
+  version: 0.1.1
+
+source:
+  url: https://files.pythonhosted.org/packages/57/f3/ba8605feae325db6d493ba604d5a07e08ff3092e83024799966986dc0c23/pybind11_rdp-0.1.1.tar.gz
+  sha256: 9d39d4054fb1d3d2f1ab60252575dd1be068f1b9d27133ee289de6aef1ddde98
+
+build:
+  number: 0
+  script: python -m pip install . -vvv
+  skip: True  # [win]
+
+requirements:
+  build:
+    - "{{ compiler('cxx') }}"
+    - cmake
+    - make  # [not win]
+    - ninja
+
+  host:
+    - python
+    - pip !=22.1.0
+
+  run:
+    - python
```

```suggestion
  run:
    - python
    - numpy
```
Missing a package from run.

https://github.com/cubao/pybind11-rdp/blob/a2c80310aed756d39f8365d91b62800db8d9d0e3/setup.py#L135